webshop-backend/src/Validator.java

```diff
@@ -0,0 +1,29 @@
+public class Validator {
+
```

Comment:
Since this is a Validator class, you don't want to make instance of this class.
You should add a private constructor with no arguments.